xbmc/pvr/windows/GUIWindowPVRBase.h

```diff
@@ -112,6 +115,12 @@ namespace PVR
   private:
     bool OpenChannelGroupSelectionDialog(void);
 
+    bool HasChannelGroupsControl();
+    bool IsChannelGroupsControlFocused();
+    void InitChannelGroupsControl();
+    bool SelectActiveChannelGroup();
+    bool ActivateSelectedChannelGroup();
```

Comment:
and here `ChannelGroupsControlSetSelectedGroup`

Reply:
No. ;-)